src/test/java/com/cardinity/CardinityIntegrationTest.java

```diff
@@ -121,6 +122,30 @@ public void testCreateDeclinedPayment() {
         assertFalse(resultPayment.isThreedsV1());
     }
 
+    @Test
+    public void testCreateApprovedPaymentWithDescriptorSuffix() {
+        Payment payment = getBaseCCPayment();
+        payment.setStatementDescriptorSuffix("Suffix");
+        Result<Payment> initialResult = client.createPayment(payment);
+        assertTrue(initialResult.isValid());
+        Payment resultPayment = initialResult.getItem();
+        assertEquals(APPROVED, resultPayment.getStatus());
+        assertNotNull(resultPayment.getId());
+        assertThat(resultPayment.getPaymentInstrument(), instanceOf(Card.class));
+        assertFalse(resultPayment.getLive());
+        assertEquals(TEST_PAYMENT_DESCRIPTION, resultPayment.getDescription());
+        assertFalse(resultPayment.isThreedsV2());
+        assertFalse(resultPayment.isThreedsV1());
+    }
+
+    @Test(expected = ValidationException.class)
+    public void testCreateDeclinedPaymentWithDescriptorSuffix() {
```

Comment:
This test is pretty much a duplicate of a test PaymentValidatorTest.testValidateWrongStatementDescriptorSuffix . This test is not testing integration. Also part of test is not never executed. `assertTrue(initialResult.isValid());`  is never executed because createPayment() method throws ValidationException. Would remove such test

Reply:
Resolved 
https://github.com/cardinity/cardinity-sdk-java/pull/3/commits/5ff6e03cdbe6b5676919a77ff9410a38078739ff